server/auth/registrar.go

```diff
@@ -301,12 +301,114 @@ func (auth *AuthManager) handlePostBond(conn comms.Link, msg *msgjson.Message) *
 	return nil
 }
 
+func (auth *AuthManager) sendPostBondRes(conn comms.Link, reqID uint64, postBondRes *msgjson.PostBondResult, acctID account.AccountID) {
+	resp, err := msgjson.NewResponse(reqID, postBondRes, nil)
+	if err != nil { // shouldn't be possible
+		return
+	}
+	err = conn.Send(resp)
+	if err != nil {
+		log.Warnf("Error sending postbond result to user %v: %v", acctID, err)
+		if err = auth.Send(acctID, resp); err != nil {
+			log.Warnf("Error sending feepaid notification to account %v: %v", acctID, err)
+			// The user will need to either 'connect' to see confirmed status,
+			// or postbond again. If they reconnected before it was confirmed,
+			// they must retry postbond until it confirms and is added to the DB
+			// with their new account.
+		}
+	}
+}
+
+// storeUpdaterBondsAndRespond is called for assets that support updating of
+// bonds. Since previous bonds that were posted may have been updated by the
+// new bond that is currently being posted, we need to check the blockchain
+// for all of the bonds that are currently active for the account and determine
+// the user's tier based on them.
```

Comment:
What if the user just does not tell the server that it updated its bonds? I'm trying to reason about it:

Client makes two bonds with one account. The locktimes are far apart. They tell the server and the longer locktime has a bunch in it so they have a strong bond. They then use the shorter locktime as a change bond and drain most of the value. They don't tell the server so it still thinks there is a strong bond, but we made it much weaker with a longer locktime. Is this not possible? The user now has a lot of bond on paper but they siphoned it out on chain. The server wouldn't know unless it checks every block.

Reply:
After some more thinking I believe that the contract does prevent value to be extracted.